### ISSUE TITLE
Repository settings

### ISSUE THREAD

**Issue by olenagerasimova** [artipie/rpm-adapter#275]
Naming policy, digest algorithm and filelists for rpm repository are configured on storage level. We need to accept these settings in `RpmUpload` and use them to create `Rpm` accordingly.

**Comment by olenagerasimova** [comment]
@0crat assign me

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/artipie/rpm-adapter/issues/275#issuecomment-646525801))

@olenagerasimova The job [#275](https://github.com/artipie/rpm-adapter/issues/275) assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/6e1c9569-ee9d-4b76-99a6-3272294c6182); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@592e6706-44da-4634-a6e2-8ab630bff16f, thread: PQ-CT2E6TK9B
com.jcabi.xml.StrictXML[124] java.lang.IllegalArgumentException: 2 error(s) in XML document: -1:-1: cvc-pattern-valid: Value 'gh:Week 26' is not facet-valid with respect to pattern '[a-z]{2}:[A-Z0-9a-z.\-#/]+' for type 'milestoneID'.;-1:-1: cvc-complex-type.2.2: Element 'successor' must have no element [children], and the value must be valid.
```

[0.53.15](https://github.com/zerocracy/farm): CID: [92ffd047-a48d-4ac4-ae21-5ae21bfc7b76](https://www.0crat.com/footprint/CT2E6TK9B/92ffd047-a48d-4ac4-ae21-5ae21bfc7b76), Type: "Job milestoned"

**Comment by 0pdd** [comment]
@olenagerasimova 2 puzzles [#281](https://github.com/artipie/rpm-adapter/issues/281), [#282](https://github.com/artipie/rpm-adapter/issues/282) are still not solved.

**Comment by olenagerasimova** [comment]
#276 merged

**Comment by 0pdd** [comment]
@olenagerasimova 4 puzzles [#281](https://github.com/artipie/rpm-adapter/issues/281), [#282](https://github.com/artipie/rpm-adapter/issues/282), [#286](https://github.com/artipie/rpm-adapter/issues/286), [#287](https://github.com/artipie/rpm-adapter/issues/287) are still not solved.

**Comment by 0pdd** [comment]
@olenagerasimova 3 puzzles [#282](https://github.com/artipie/rpm-adapter/issues/282), [#286](https://github.com/artipie/rpm-adapter/issues/286), [#287](https://github.com/artipie/rpm-adapter/issues/287) are still not solved; solved: [#281](https://github.com/artipie/rpm-adapter/issues/281).

**Comment by 0pdd** [comment]
@olenagerasimova 2 puzzles [#282](https://github.com/artipie/rpm-adapter/issues/282), [#287](https://github.com/artipie/rpm-adapter/issues/287) are still not solved; solved: [#281](https://github.com/artipie/rpm-adapter/issues/281), [#286](https://github.com/artipie/rpm-adapter/issues/286).

**Comment by 0pdd** [comment]
@olenagerasimova the puzzle [#282](https://github.com/artipie/rpm-adapter/issues/282) is still not solved; solved: [#281](https://github.com/artipie/rpm-adapter/issues/281), [#286](https://github.com/artipie/rpm-adapter/issues/286), [#287](https://github.com/artipie/rpm-adapter/issues/287).

**Comment by 0pdd** [comment]
@olenagerasimova all 4 puzzles are solved here: [#281](https://github.com/artipie/rpm-adapter/issues/281), [#282](https://github.com/artipie/rpm-adapter/issues/282), [#286](https://github.com/artipie/rpm-adapter/issues/286), [#287](https://github.com/artipie/rpm-adapter/issues/287).
